tests/user/test_models.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from django.contrib.auth import get_user_model
+
+User = get_user_model()
+
+pytestmark=pytest.mark.django_db
+
+
+class TestUserModel:
+
+    def test_str_return(self, user_factory):
+        user = user_factory(email="shah@gmail.com")
+        assert user.__str__() == "shah@gmail.com"
+    
+    def test_valid_phone(self,user_factory):
+        try:
+            user = user_factory(phone="0000000")
+        except Exception as e:
+            pytest.fail(f"Exeption raised {type(e).__name__}")
+    
+    def test_invalid_phone1(self, user_factory):
+        with pytest.raises(Exception):
+            user = user_factory(phone="+994700000")
+
+    def test_invalid_phone2(self, user_factory):
+        with pytest.raises(Exception):
+            user = user_factory(phone="shahan")
+    
+    def test_valid_fin_code(self, user_factory):
+        try:
```

Comment:
No this way. https://stackoverflow.com/a/68012715, use does_not_raise, this way it is better looking